packs/spell-effects/spell-effect-mantle-of-the-melting-heart.json

```
@@ -0,0 +1,228 @@
+                                "not": "melting-heart-first:golden-flesh"
+                            }
+                        ],
+                        "value": "golden-flesh"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.Label.ReactiveTouch",
+                        "predicate": [
+                            {
+                                "not": "melting-heart-first:reactive-touch"
+                            }
+                        ],
+                        "value": "reactive-touch"
+                    },
+                    {
+                        "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.Label.WeightedGrasp",
+                        "predicate": [
+                            {
+                                "not": "melting-heart-first:weighted-grasp"
+                            }
+                        ],
+                        "value": "weighted-grasp"
+                    }
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "Resistance",
+                "predicate": [
+                    {
+                        "or": [
+                            "melting-heart-first:copper-core",
+                            "melting-heart-second:copper-core"
+                        ]
+                    }
+                ],
+                "type": "electricity",
+                "value": 10
+            },
+            {
+                "key": "Immunity",
+                "predicate": [
+                    {
+                        "or": [
+                            "melting-heart-first:golden-flesh",
+                            "melting-heart-second:golden-flesh"
+                        ]
+                    }
+                ],
+                "type": [
+                    "disease",
+                    "poison"
+                ]
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "against-poison-disease-onself",
+                    {
+                        "or": [
+                            "melting-heart-first:golden-flesh",
+                            "melting-heart-second:golden-flesh"
+                        ]
+                    }
+                ],
+                "selector": [
+                    "saving-throw",
+                    "skill-check"
+                ],
+                "value": 2
+            },
+            {
+                "category": "unarmed",
+                "damage": {
+                    "base": {
+                        "damageType": "bludgeoning",
+                        "dice": 1,
+                        "die": "d4"
+                    }
+                },
+                "group": "flail",
+                "key": "Strike",
+                "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.CabledFist.Label",
```

```suggestion
                "label": "PF2E.SpecificRule.SpellEffectMantle.MeltingHeart.CabledFist.Label",
```
Doesn't look like this localization string exists yet?

---

I forgot to hit save when I made the change.